modules/HardwareDrivers/NfcReaders/NxpNfcFrontendTokenProvider/main/auth_token_providerImpl.cpp

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020 - 2025 Pionix GmbH and Contributors to EVerest
+
+#include "auth_token_providerImpl.hpp"
+#include <iomanip>
+#include <sstream>
+
+namespace module {
+namespace main {
+
+static std::string nfcidToString(const std::vector<std::uint8_t>& nfcid) {
+    std::ostringstream oss;
+
+    for (std::uint16_t byte : nfcid) {
+        oss << std::setfill('0') << std::setw(2) << std::uppercase << std::hex << byte;
+    }
+
+    return oss.str();
+}
+
+void auth_token_providerImpl::detected_rfid_token_callback(
+    const std::pair<std::string, std::vector<std::uint8_t>>& reply) {
+    auto& [protocol, nfcid] = reply;
+
+    const auto now = std::chrono::steady_clock::now();
+    const auto debounce_interval = std::chrono::milliseconds(config.token_debounce_interval_ms);
+
+    types::authorization::ProvidedIdToken provided_token;
+
+    try {
+        provided_token.id_token = {nfcidToString(nfcid), types::authorization::string_to_id_token_type(protocol)};
+    } catch (const std::out_of_range& e) {
+        return;
+    }
+
+    provided_token.authorization_type = types::authorization::AuthorizationType::RFID;
+
+    if (now < last_rfid_submit + debounce_interval) {
+        if (config.debug) {
+            EVLOG_info << "Ignoring rfid/nfc token (debouncing): " << protocol << ": " << nfcidToString(nfcid);
```

Comment:
```suggestion
            EVLOG_debug << "Ignoring rfid/nfc token (debouncing): " << protocol << ": " << nfcidToString(nfcid);
```

Whats your opinion on using this? Then you can selectively look at debug output by modifying /etc/everest/default_logging.cfg, if needed.